examples/scripts/spm_descent.py

```diff
@@ -37,9 +37,12 @@
 problem = pybop.FittingProblem(model, parameters, dataset)
 cost = pybop.SumSquaredError(problem)
 optim = pybop.Optimisation(
-    cost, optimiser=pybop.GradientDescent, sigma0=0.022, verbose=True
+    cost,
+    optimiser=pybop.GradientDescent,
+    sigma0=0.022,
```

Comment:
This a good chance to align with results from: https://github.com/pybop-team/PyBOP/blob/a6461da16b985cb8bf69b62219e2d6835e881afd/examples/notebooks/optimiser_calibration.ipynb
```suggestion
    sigma0=0.011,
```